PowerEditor/installer/themes/Bespin.xml

```diff
@@ -1,46 +1,46 @@
 <?xml version="1.0" encoding="UTF-8" ?>
-<!--//

```

Comment:
Every comments seems to have `LF` EOL and rest of file have `CR LF`. This should be fixed to not have mixed EOL in files.